.github/workflows/stockfish_test.yml

```diff
@@ -115,7 +115,7 @@ jobs:
 
       - name: Extract the bench number from the commit history
         run: |
-          git log HEAD | grep "\b[Bb]ench[ :]\+[0-9]\{7\}" | head -n 1 | sed "s/[^0-9]*\([0-9]*\).*/\1/g" > git_sig
+          git log HEAD | grep -o "\b[Bb]ench[ :]\+[1-9][0-9]\{5,9\}\b" | head -n 1 | sed "s/[^0-9]//g" > git_sig
```

Comment:
`sed "s/[^0-9]//g"` can be replaced by `grep -o "[0-9]\+"`. I think the latter is easier to read because you only have to know what the `grep` command does.

Reply:
sorry, this review crossed my merging..

Reply:
@vondele did you miss this or you think it wasn't relevant?

Reply:
vondele was already in the process of merging when I submitted my comments. Both comments were just suggestions, so they can be ignored.

Reply:
Although subjective, this might be easier to read.
`git log HEAD | grep -oi "bench[ :]\+[1-9][0-9]\{5,9\}\b" | grep -m 1 -o "[0-9]*" > git_sig`

1. Adding -i option to grep to make the search case-insensitive, so "Bench" and "bench" are both matched. 
2. The head -n 1 part is replaced by adding -m 1 to the second grep command, which stops reading the input after finding the first match.
3. The sed "s/[^0-9]//g" command is replaced by a second grep command. This grep command uses the -o option to print only the matched part of the line and finds the numeric sequence in the pattern. This way, the requirement to understand the sed command is eliminated, making the command easier to comprehend for readers who only know grep.